### PR TITLE
Release '0.11.0'

### PR DESCRIPTION
**Pull request by piotrekno1**
Would like to release avro schema validation feature to the world!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +116,44 @@ from djangoevents import subscribe_to                   # from eventsourcing.dom
 from djangoevents import EventSourcedRepository         # from eventsourcing.infrastructure.event_sourced_repo import EventSourcedRepository
 ```
 
+### Documenting event schema
```
Comment (renou) [inline README.md]: maybe add some documentation of the meaning and how to use the abstract decorator?

File: README.md [hunk 2]
```diff
@@ -109,6 +116,44 @@ from djangoevents import subscribe_to                   # from eventsourcing.dom
...
+$ tree project
+|- src
+|--- manage.py
+|--- ..
+|-avro
+|--- aggragate_name/
+|----- aggregate_name_test_event1_v1.json
+|----- aggregate_name_test_event2_v1.json
+...
+```
```
Comment (renou) [inline README.md]: Maybe we add a small example of avro json?

**Comment by piotrekno1** [comment]
@renou Done. Wating for CI and merging.

**Comment by renou** [comment]
:1st_place_medal:

**Pull request merged**